packages/packagers/js/src/prelude.js

```diff
@@ -6,10 +6,24 @@
 // anything defined in a previous bundle is accessed via the
 // orig method which is the require for previous bundles
 
-// eslint-disable-next-line no-global-assign
-parcelRequire = function(modules, cache, entry, globalName) {
+(function(modules, cache, entry, globalName) {
+  /* eslint-disable no-undef */
+  var globalObject =
+    typeof globalThis !== 'undefined'
+      ? globalThis
+      : typeof self !== 'undefined'
+      ? self
+      : typeof window !== 'undefined'
+      ? window
+      : typeof global !== 'undefined'
+      ? global
+      : {};
```

Comment:
😭

Reply:
Runnning out of options for a global object here, async loading wouldn't be supported in this case 🙈

Reply:
lol strict mode is annoying. Ideally we'd get rid of the global variable entirely... not sure how though.

Reply:
we could do it based on target maybe. i.e. browser target has `window`, node has `global`, etc.

Reply:
I don't see a way of getting the modules/state from an earlier `parcelRequire` call without globals.
Unless, loading an async module could somehow happen inside the closure of the initial `parcelRequire`, passing it over. But with the `<script>` injection that doesn't seem possible.